### ISSUE TITLE
Combustion Generator on Wrong Power grid (same as Windmill and Waterwheel) should be on same one as Steam generator 6.3.1

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#3331]
I have 3 Combustion Generator's outputting 1000w each
and 2 Steam Generator's Outputting 200w each
![snap965](https://user-images.githubusercontent.com/29383891/34751724-35e2cab0-f5a6-11e7-8a37-7862eb683888.png)
as you can see above the Power grid is 3200w 
![snap973](https://user-images.githubusercontent.com/29383891/34752416-3213ae0a-f5aa-11e7-8061-655ad73cef72.png)
just worked out why it was saying Extra 200w that is what the windmill is outputting
![snap972](https://user-images.githubusercontent.com/29383891/34752309-57e7e7d2-f5a9-11e7-90f3-c3ec636221d0.png)
Power grid above from Steam generators is 400w with Pumpjack using 100w)
![snap971](https://user-images.githubusercontent.com/29383891/34751968-7209af76-f5a7-11e7-95da-2dbc3f1a7c8c.png)
My pumpjack is getting its power requirment from the Steam generators with no problem
![snap970](https://user-images.githubusercontent.com/29383891/34752383-faee682a-f5a9-11e7-91b8-76ffbd433c37.png)
but Factory can ONLY see the Steam Generator's power grid
![snap969](https://user-images.githubusercontent.com/29383891/34752474-7719e208-f5aa-11e7-92e1-35f99106642c.png)
as you can see Factory is Right next to the Combustion Generator's.

**Comment by Conchipedia** [comment]
All i can work out by this is that the Combustion Generator is on the Wrong power grid, it is on the same one for Windmill and waterwheel

**Comment by Conchipedia** [comment]
Steam Generator (cannot see any other storage or this storage be seen from elsewhere) 6.2.0 #3146

Because @MarjieAVolk said that "Steam Generator" is getting removed in the above linked Issue then this problem with Combustion Generator's will be a **game breaker** as the **only** thing that is powering the Factory is Steam Generator and that needs 2500w to craft

**Comment by MarjieAVolk** [comment]
Hm this sort of thing is both hard to understand and hard to debug right now, because of all the automatic linking based on distance, and the different objects having different radii and whatnot.  I'm also having trouble visualizing where all these objects are in relation to each other.  The steam generators and windmill are a floor above the factory and combustion generators?

@Conchipedia For now I'd recommend picking things up and putting them down and fiddling with it, see if you can get them all to connect.

**Comment by Conchipedia** [comment]
i moved the Steam Generators from below as they were next to pumping jack now they are right next to Combustion Generators (pumpjack can still see the power grid), also i moved the factory around also A combustion generator (wasted Petroleum doing that) and the Windmill is on the surface
![snap974 edited](https://user-images.githubusercontent.com/29383891/34791424-d851dab6-f63c-11e7-8c2d-8eeeb2508148.png)
the only power grid the Factory can see is the the Steam generator one and no matter where i place Combustion Generators they still only belong to the Power Grid that has windmill in it.
![snap975](https://user-images.githubusercontent.com/29383891/34791580-4946240c-f63d-11e7-883f-3830b3e6346a.png)
and as you can see from the Mill power Tab it can also see windmill and 3 Combustion generators power Grid (200w = Windmill 3000w = 2 x Combustion Generators (1000w each).

In conclusion, no matter where i place the Combustion Generators (above Factory, below Factory, next to Factory) the Factory will Only see the Power Grid of the Steam Generator's (which as far as i can tell is a totally different method of power than what is generated by the Windmill and Waterwheel) so Combustion Generator has been added to the **wrong** power grid

**Comment by MarjieAVolk** [comment]
Ohhhh right they are different kinds of power, I forgot about that >>  Yeah okay, so the power generators and power types are being consolidated and cleaned up right now, so hopefully that will fix this issue.

**Comment by Conchipedia** [comment]
just think of me as that little voice inside your head reminding you about things you already knew ❤️

**Comment by MarjieAVolk** [comment]
Okay so closing this as intended behavior for now.  Power types will be consolidated in the next release.  And I will also think about how to make this easier to understand.

**Comment by Conchipedia** [comment]
I can now play using Combustion Generators, Admin did a quick edit so  instead of mechanical energy now it makes electrical energy
